yfinance/base.py

```diff
@@ -616,6 +616,11 @@ def _get_info(self, proxy=None):
         ticker_url = "{}/{}".format(self._scrape_url, self.ticker)
 
         # get info and sustainability
+        json_data = utils.get_json_data_stores(ticker_url, proxy, self.session)
```

Comment:
This function call can fail, need to handle IndexError
`json_str = html.split('root.App.main =')[1].split(`
> IndexError: list index out of range